### PR TITLE
add Analytics tab

### PR DESCRIPTION
**Pull request by uafio**
currently container a table of solves sorted by date of solve in reverse

### CODE REVIEW

File: ctf_framework/views/analytics.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+from ..models import Solve
+
+
+@login_required()
+def index(request):
+    """List all active Categories."""
+    if not request.user.is_staff:
+        return HttpResponseForbidden()
+
+    solves = Solve.objects.all().reverse()#[:1000] # last 1000 entries
```
Comment (Corb3nik) [inline ctf_framework/views/analytics.py]: Remove the `#[:1000] # last 1000 entries` comment :)

File: static/css/main.css [hunk 2]
```diff
@@ -6,3 +6,69 @@
 .link-success a {
   color: white;
 }
+
+table.dataTable thead>tr>td.sorting,
```
Comment (Corb3nik) [inline static/css/main.css]: Where did you get all of these lines?

`main.css` is for custom stuff, but this seems related to the `DataTable` plugin

File: ctf_framework/templates/analytics/index.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{% extends "layouts/base.html" %}
+{% block content %}
+
+<div class="container">
+    <table id="dtVerticalScrollExample" scrollY="100px" class="table table-striped table-bordered table-sm" cellspacing="0" width="100%">
```
Comment (Corb3nik) [inline ctf_framework/templates/analytics/index.html]: We should rename the `id` to something relevant.

File: static/css/main.css (hunk 2 above)
Comment (uafio) [reply static/css/main.css]: https://mdbootstrap.com/docs/jquery/tables/scroll/

File: ctf_framework/templates/analytics/index.html (hunk 3 above)
Comment (uafio) [reply ctf_framework/templates/analytics/index.html]: will do

**Pull request merged**